apm-sniffer/apm-sdk-plugin/druid-1.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/druid/v1/define/DruidDataSourceInstrumentation.java

```diff
@@ -94,4 +109,10 @@ public boolean isOverrideArgs() {
     public StaticMethodsInterceptPoint[] getStaticMethodsInterceptPoints() {
         return new StaticMethodsInterceptPoint[0];
     }
+
+    @Override
+    public boolean isBootstrapInstrumentation() {
+        return true;
```

Comment:
Same as above

Reply:
Without rewriting, enhancement does not work